src/renderers/shared/stack/event/EventConstants.js

```diff
@@ -87,7 +87,9 @@ var topLevelTypes = keyMirror({
   topVolumeChange: null,
   topWaiting: null,
   topWheel: null,
-});
+};
+
+export type TopLevelTypes = $Enum<typeof topLevelTypes>;
```

Comment:
@vjeux @gaearon I took a stab at documenting this: https://github.com/facebook/flow/pull/2465

By the way, judging by a comment in the source code, it looks like `$Keys` is preferred to `$Enum`, which should be removed: https://github.com/facebook/flow/blob/master/src/typing/type_annotation.ml#L247